go/tools/microbench/results_test.go

```diff
@@ -234,4 +234,25 @@ func TestMergeMicroBenchmarkDetails(t *testing.T) {
 			c.Assert(got, qt.DeepEquals, tt.want)
 		})
 	}
-}
\ No newline at end of file
+}
+
+func TestHumanReadableStrings(t *testing.T) {
```

Comment:
I've got this test failing here:

![image](https://user-images.githubusercontent.com/35779988/112659866-1b850980-8e55-11eb-9b37-894eebc44e14.png)

```
go version go1.16 darwin/amd64
```